styles.css

```diff
@@ -0,0 +1,335 @@
+  color: #444;
+}
+
+main a:hover,
+main a:focus {
+  color: black;
+}
+
+/* ==============
+   DEMO STYLES
+   ============== */
+
+.site-header {
+  position: relative;
+}
+
+.site-header h1 {
+  display: inline-block;
+  line-height: 112px;
+  margin: 0;
+  position: relative;
+  text-align: center;
+  width: 120px;
+}
+
+.site-header .logo {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+}
+
+.menu-icon__line {
+  background-color: silver;
+  height: 2px;
+  margin: 8px auto;
+  transition: var(--transition-default);
+  width: 42px;
+}
+
+[aria-expanded="true"] .menu-icon__line:nth-child(1) {
+  transform: translateY(10px) rotate(45deg);
+}
+
+[aria-expanded="true"] .menu-icon__line:nth-child(2) {
+  opacity: 0;
+}
+
+[aria-expanded="true"] .menu-icon__line:nth-child(3) {
+  transform: translateY(-10px) rotate(-45deg);
+}
+
+/* ==============
+   MEDIA QUERIES
+   ============== */
+
+
+
+@media (max-width: 599px) {
+  .site-nav {
+    position: absolute;
+    top: 0;
+    right: 0;
+    text-align: right;
+    z-index: 1
+  }
+}
+  
+  @media (max-width:999px) {
+  .speaker-details img {
+  float: left;
+  margin-right: 18px;
+    width:33%;
+    
+}
+}
+  
+@media (min-width:1000px) {
+  .speaker-details {
+    display: inline-block;
+
+    vertical-align: top;
+    width:33%;
+  }
+  
+  
+  .speaker-details img {
+    display: block;
+    margin: auto;
+    padding-top: 0;
+  }
+}
+
+  .menu-trigger {
+    background-color: #141518;
+    color: silver;
+    padding: var(--spacing-sm);
+    transition: var(--transition-default);
+  }
+
+
+  .menu-trigger:hover,
+  .menu-trigger:focus {
+    background-color: black;
+    color: white;
+  }
+
+  .menu-trigger[aria-expanded=true] {
+    background-color: transparent;
+    opacity: 0.72;
+  }
+
+  .menu-trigger[aria-expanded=true]:hover,
+  .menu-trigger[aria-expanded=true]:focus {
+    opacity: 1;
+  }
+
+  .menu-trigger:hover .menu-icon__line,
+  .menu-trigger:focus .menu-icon__line {
+    background-color: white;
+  }
+
+  .menu-trigger:not([aria-expanded]) {
+    display: none;
+  }
+
+  .menu-trigger[aria-expanded=false]+.menu-list {
+    display: none;
+  }
+
+  .menu-list {
+    animation: slideIn 420ms cubic-bezier(.68, -0.55, .27, 1.55) 1;
+    background-color: #141518;
+    padding-top: 120px;
+    position: absolute;
+    top: calc(var(--spacing-md) * -1);
+    right: calc(var(--spacing-md) * -1);
+    transform-origin: top;
+    width: 65vw;
+    z-index: -1;
+  }
+
+  .menu-list.full-screen {
+    animation-duration: 480ms;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    width: unset;
+  }
+
+  .menu-list a {
+    color: silver;
+    display: block;
+    font-size: 1.3em;
+    padding: var(--spacing-md) var(--spacing-lg);
+    text-align: center;
+    text-decoration: none;
+    transition: var(--transition-default);
+  }
+
+  .menu-list a:hover,
+  .menu-list a:focus {
+    background-color: black;
+    color: white;
+  }
+}
```

Comment:
This is an extra `}`, which Glitch points out to you if you use the "Format This File" button on your CSS, and it results in the rules on lines 227 and 231 not being applied as expected.   
```suggestion
```